extensions/amp-story-interactive/0.1/test/test-amp-story-interactive-slider.js

```diff
@@ -146,5 +146,15 @@ describes.realWin(
         'i-amphtml-story-interactive-post-selection'
       );
     });
+
+    it.only('should display the emoji in the bubble when the user drags the slider', async () => {
```

Comment:
Test name should be "should display the emoji in the bubble from the attribute configuration". Also, remove the `.only`